packages/types/index.ts

```diff
@@ -13,10 +13,23 @@ export interface Context {
   sourceFileExtension: 'js' | 'ts'
 }
 
+export interface Options {
+  /** Overrides for layer names */
+  layerNames: Record<string, string>
+  additionalSegmentNames: Array<string>
+}
```

Comment:
issue: this is FSD-specific stuff, I don't think it belongs to Steiger core

Reply:
Yes, you're right. No problem, I'll remove it. I added that as a placeholder anyway, they are not supported yet.